src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract String draw();

```

Comment:
<img width="1042" alt="Screen Shot 2022-09-20 at 6 22 14 PM" src="https://user-images.githubusercontent.com/90692308/191298731-54ae6781-6652-4612-90fe-1a63d9aea3de.png">
There must also be a separate interface for the draw method